ci/pipelines/infra-drift-detector-lower-envs.yml

```diff
@@ -192,7 +192,7 @@ jobs:
         put: slack-notification
         attempts: 10
         params:
-          channel: '#govuk-pay-starling'
+          channel: '#govuk-pay-activity'
```

Comment:
I think failures should definitely go to `#govuk-pay-starling`, otherwise we'll never see them in the `activity` channel.